test/build/index.js

```diff
@@ -3,6 +3,7 @@ const { expect } = require('chai');
 
 describe('build', () => {
   it('should build UMD compliant output', () => {
-    expect(ReactMobiledocEditor).to.be.an('object');
+    expect(typeof ReactMobiledocEditor).to.equal('object');
+    expect(ReactMobiledocEditor).to.be.a('module');
```

Comment:
@gpoitch This test began failing but the UMD header hasn't changed in the build output. As far as I can tell this is just a difference in how the Chai matcher reports this. I don't _think_ there's a breaking change here -- but I'm tagging you in case you know otherwise.

Reply:
look ok to me